collision.cpp

```diff
@@ -2,422 +2,444 @@
+		b = 2.0f * (glm::dot(velocity, (basePos - tri.b.pos)));
+		c = glm::distance2(tri.b.pos, basePos) - 1.0f;
+		// Find smallest solution, if available
+		if (GetSmallestRoot(a, b, c, t, &newT)) {
+			t = newT;
+			foundCollision = true;
 			hitPoint = tri.b.pos;
-        }
-
-        // Check point C
-        b = 2.0f * ( glm::dot(velocity, (basePos - tri.c.pos)) );
-        c = glm::distance2(tri.c.pos, basePos) - 1.0f;
-        // Find smallest solution, if available
-        if (GetSmallestRoot(a, b, c, t, &newT)) {
-            t = newT;
-            foundCollision = true;
-			hitPoint = tri.c.pos;	
-        }
+		}
+
+		// Check point C
+		b = 2.0f * (glm::dot(velocity, (basePos - tri.c.pos)));
+		c = glm::distance2(tri.c.pos, basePos) - 1.0f;
+		// Find smallest solution, if available
+		if (GetSmallestRoot(a, b, c, t, &newT)) {
+			t = newT;
+			foundCollision = true;
+			hitPoint = tri.c.pos;
+		}
 
 		// Check sphere against tri's line-segments
 
-		if ( CheckSweptSphereVsLinesegment(tri.a.pos, tri.b.pos, basePos, velocity, t, &newT, &hitPoint) ) {
+		if (CheckSweptSphereVsLinesegment(tri.a.pos, tri.b.pos, basePos,
+										  velocity, t, &newT, &hitPoint)) {
 			foundCollision = true;
 			t = newT;
 		}
 
-		if ( CheckSweptSphereVsLinesegment(tri.b.pos, tri.c.pos, basePos, velocity, t, &newT, &hitPoint) ) {
+		if (CheckSweptSphereVsLinesegment(tri.b.pos, tri.c.pos, basePos,
+										  velocity, t, &newT, &hitPoint)) {
 			foundCollision = true;
 			t = newT;
 		}
 
-		if ( CheckSweptSphereVsLinesegment(tri.c.pos, tri.a.pos, basePos, velocity, t, &newT, &hitPoint) ) {
+		if (CheckSweptSphereVsLinesegment(tri.c.pos, tri.a.pos, basePos,
+										  velocity, t, &newT, &hitPoint)) {
 			foundCollision = true;
 			t = newT;
 		}
 	}
 
 	if (foundCollision) {
-		float distanceToHitpoint = t * glm::length( velocity ); // furthest the sphere can travel along its velocity vector until collision happens
-		if ( !ci->didCollide || (ci->nearestDistance > distanceToHitpoint) ) {
+		float distanceToHitpoint =
+			t *
+			glm::length(velocity); // furthest the sphere can travel along its
+								   // velocity vector until collision happens
+		if (!ci->didCollide || (ci->nearestDistance > distanceToHitpoint)) {
 			ci->didCollide = true;
 			ci->nearestDistance = distanceToHitpoint;
 			ci->hitPoint = hitPoint;
 			if (embeddedInPlane) {
-				//ci->nearestDistance = -VERY_CLOSE_DIST;
-				//ci->hitPoint -= VERY_CLOSE_DIST*p.normal;
+				// ci->nearestDistance = -VERY_CLOSE_DIST;
+				// ci->hitPoint -= VERY_CLOSE_DIST*p.normal;
 			}
 		}
 	}
-
 }
 
-CollisionInfo CollideEllipsoidWithTriPlane(EllipsoidCollider ec, glm::vec3 velocity, glm::vec3 gravity, TriPlane* triPlanes, int triPlaneCount)
-{
-    // Convert to ellipsoid space
+CollisionInfo CollideEllipsoidWithTriPlane(EllipsoidCollider ec,
+										   glm::vec3 velocity,
+										   glm::vec3 gravity,
+										   TriPlane *triPlanes,
+										   int triPlaneCount) {
```

Comment:
we can setup a different line length if this seems to narrow

Reply:
Yes, I think that should be around 120 columns. But even then, I dislike if I'm forced into multi-line in e.g. a comment because of a few characters (that's bound to happen at any columns setting though).

Reply:
changed it to 120 lines in the other pr